<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="frameLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>校园活动报名与签到</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 18px; font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="userLabel">
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="usernameEdit">
          <property name="placeholderText">
           <string>admin / host / alice</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="passLabel">
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="passwordEdit">
          <property name="placeholderText">
           <string>admin123 ...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="roleLabel">
          <property name="text">
           <string>角色</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="roleCombo">
          <item>
           <property name="text">
            <string>管理员</string>
           </property>
           <property name="userData">
            <string notr="true">admin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>发起人</string>
           </property>
           <property name="userData">
            <string notr="true">initiator</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学生</string>
           </property>
           <property name="userData">
            <string notr="true">student</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>登录</string>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:8px; font-weight:bold;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registerButton">
          <property name="text">
           <string>创建/注册账号</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="hintLabel">
        <property name="text">
         <string>提示: 管理员 admin/admin123; 发起人 host/host123; 学生 alice/alice123, bob/bob123</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #555;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

